specification/ai/HealthInsights/HealthInsights.OncoPhenotype/tspconfig.yaml

```diff
@@ -1,19 +1,51 @@
-emit: []
+parameters:
+  "service-dir":
+    default: "sdk/healthinsights"
+  "service-name":
+    default: "HealthInsights"
+  "dependencies":
+    "additionalDirectories":
+      - "specification/ai/HealthInsights/HealthInsights.Common/"
+      - "specification/ai/HealthInsights/HealthInsights.OpenAPI/"
```

Comment:
Invalid path given the move which is failing your sdk generation step.

Reply:
Thanks for catching that, updated it for the team in all instances